libs/constrained-generators/src/Constrained/Spec/Map.hs

```diff
@@ -271,12 +278,40 @@ instance BaseUniverse fn => Functions (MapFn fn) fn where
                       Set.empty
                       must
                       size
-                      (constrained $ \kv -> satisfies (app (sndFn @fn) (app (toGenericFn @fn) kv)) elemspec)
+                      (constrained $ \kv -> satisfies (snd_ kv) elemspec)
                       foldspec
+                      Nothing
                 -- NOTE: you'd think `MemberSpec [r]` was a safe and easy case. However, that requires not only that the elements
                 -- of the map are fixed to what is in `r`, but they appear in the order that they are in `r`. That's
                 -- very difficult to achieve!
                 _ -> ErrorSpec ["Rng on bad map spec", show spec]
+    RngSet ->
```

Comment:
What would we need to add elsewhere to be able to use `fromList_ . rng_` instead of `rngSet_` to make things more general?

Reply:
I think this just works.